### DISCUSSION TITLE
Join project requests

### DISCUSSION THREAD

**Discussion by AksonovSergei** [KPI-Rover/ecu_sw#11] [General]
Якщо ви зацікавлені взяти участь у проєкті, напишіть про це в цій темі.
Вкажіть напрямок, яким ви хотіли б займатися.
Ось список наших напрямків:
1. Розробка електроніки
2. Конструкція та 3D-друк
3. Розробка вбудованого програмного забезпечення
4. Програмне забезпечення для Raspberry Pi з використанням ROS2
6. Підготовка документації

Після того як ви розмістите своє повідомлення, ми додамо вас до нашої команди, і ви зможете обрати собі завдання.

**Comment by vetshem69** [comment]
Я візьму участь в цьому проекті, буду розробляти плату розширення і програмування STM32.

**Comment by Dementris** [comment]
Додайте будь-ласка 4. Програмне забезпечення для Raspberry Pi з використанням ROS2

**Reply by AksonovSergei** [reply]
Я додав Вас до участників проєкту.
Приєднуйтесь завтра до нашої онлайн зустрічі.

**Comment by Glubonium** [comment]
Це Загородній Владислав. Буду працювати з платою STM32 у CubeIDE.

**Reply by AksonovSergei** [reply]
Доброго дня. Ви додані до команди студентів. Перевірте будьласка свою почтову скриньку - ви маєте отримати листа запрошення до команди KPI-Rover/students.

**Comment by taketook34** [comment]
Це Хусаінов Дмитро. Буду працюваи з платою у CUbeIDE і проектувати в Kicad.

**Reply by AksonovSergei** [reply]
Доброго дня. Ви додані до команди студентів. Перевірте будьласка свою почтову скриньку - ви маєте отримати листа запрошення до команди KPI-Rover/students.

**Comment by Rhinemann** [comment]
Швед Андрій, розробка плати.

**Comment by nazarbychkar** [comment]
Добрий вечір, це Назар Бичкар, на розробку плати.

**Comment by Ryabokon-Varya** [comment]
Добрий вечір, Рябоконь Варя, розробка електроніки.

**Comment by AksonovSergei** [comment]
Привіт
@Rhinemann, @nazarbychkar, @Ryabokon-Varya 
Я додав вас до проєкту. Вам має прийти лист з проханням прийняти запрошення до організації.

Перед початком виконання задань пропоную:
1. Встановити KiCAD
2. Трохи його вивчити:
 - https://docs.kicad.org/#_getting_started
 - https://www.youtube.com/playlist?list=PL3bNyZYHcRSUhUXUt51W6nKvxx2ORvUQB

На вихідних я створю для вас завдання - ви отримуєте повідомлення на електронну адресу.

**Comment by ktryna** [comment]
Вітаю. Демʼяненко Катерина, project manager

**Comment by IO25SavelievArtem** [comment]
Добрий день, Савельєв Артем, налаштування CI/CD на GitHub Actions.

**Comment by AksonovSergei** [comment]
Привіт @ktryna and @IO25SavelievArtem 
Я додав вас до проєкту. Вам має прийти лист з проханням прийняти запрошення до організації.

Згодом я створю для вас завдання, а поки будь ласка ознайомтеся з тим що таке ROS2.

@IO25SavelievArtem 
Ось лист проєктів для котрих буде потрібно зробити CI:
- https://github.com/KPI-Rover/ros_cspc_lidar_coin_d4a
- https://github.com/KPI-Rover/ecu_sw
- https://github.com/KPI-Rover/ecu_sw_bb
- https://github.com/KPI-Rover/ros_ws
- https://github.com/KPI-Rover/ros_sw


@ktryna, Ви можете продивитись наші існуючи завдання https://github.com/orgs/KPI-Rover/projects/4 й наш сайт https://kpi-rover.github.io/ було б цікаво почути Ваші пропозиції.

**Comment by EugenFICT** [comment]
Доброго вечора. Головатенко Євгеній. Готовий приєднатися до проекту. Напрямок - підготовка документації

**Comment by makssvelichko** [comment]
Розробка вбудованого програмного забезпечення

**Comment by koroliLich** [comment]
Добрий вечір, хотів б приєднатися до команди для розробки на STM32

**Comment by hasslesstech** [comment]
Добрий вечір, це Шмуляр Олег. Бажаю приєднатися до проєкту як розробник вбудованого програмного забезпечення для STM32
